### PR TITLE
add a userenv and update README.md

### PR DESCRIPTION
**Pull request by k-rister**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
 # crucible-ci-userenvs
-External userenvs used for crucible-ci testing
+
+External userenvs used for
+(crucible-ci)[https://github.com/perftool-incubator/crucible-ci] testing.
+
+The intent of the userenvs in this repositorry is not that they be
```
Comment (rafaelfolco) [inline README.md]: strawberrrrrry

**Review by rafaelfolco — COMMENTED** [review]
I am not sure I understand the motivation on why do we need a separate repo do test unofficial userenv...

**Comment by k-rister** [comment]
> I am not sure I understand the motivation on why do we need a separate repo do test unofficial userenv...

We currently have 2 features that we are not testing at all in CI:

1. external repository management (add, update, remove)
2. external userenvs (`rickshaw-run` has logic to search for the userenv in multiple places)

External userenvs are dependent on external repositories so they can be addressed at the same time.  This is just the first piece of resolving both of those gaps -- there will have to be updates to the integration tests action itself as well as to the userenv generator in the new `get-job-parameters` action.

**Comment by k-rister** [comment]
I guess I should add that these features are supporting some of the corner case scenarios that a subset of our users experience and thus may have non-obvious purposes.  I can elaborate more offline if necessary.

**Review by rafaelfolco — APPROVED** [review]
